src/components/PeopleFilters/PeopleFilters.tsx

```diff
@@ -0,0 +1,106 @@
+import { useSearchParams } from 'react-router-dom';
+import cn from 'classnames';
+import { SearchLink } from '../SearchLink';
+import { getSearchWith } from '../../utils/searchHelper';
+
+export const PeopleFilters = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+  const sex = searchParams.get('sex');
+  const centuriesParams = searchParams.getAll('centuries');
```

Comment:
let's move this query param names to some enum, it will help us to manipulate them easily

Reply:
still not fixed
<img width="607" alt="image" src="https://github.com/mate-academy/react_people-table-advanced/assets/70220667/bdc2a434-50ee-4deb-a263-d07181635a9a">

feel free to ask in the chat if you have some questions